apps/aehttp/priv/oas3.yaml

```diff
@@ -1544,6 +1544,35 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /debug/channels/set-delegates:
+    post:
+      tags:
+        - internal
+        - channel
+        - debug
+      operationId: PostChannelSetDelegates
+      description: 'Get a channel_set_delegates_tx transaction object'
```

Comment:
"Get"?

Reply:
This is a big topic but: since the goal of this API is to fetch a channel representation - _get_ seems appropriate. Because HTTP GET having a request body is considered a bad practice we use POST instead.